app/models/finding.rb

```diff
@@ -28,6 +28,7 @@ class Finding < ApplicationRecord
   include Findings::Display
   include Findings::Expiration
   include Findings::FinalReview
+  include Findings::FindingAnswersGenerator
```

Comment:
Le pondría algo como:
```suggestion
  include Findings::ProcessEmail
```

Reply:
impeca